docs/presets/README.md

```diff
@@ -113,7 +113,7 @@ format = '\[[$symbol($profile)(\($region\))(\[$duration\])]($style)\]'
 format = '\[[$symbol($version)]($style)\]'
 
 [cmd_duration]
-format = "[⏱ $duration]($style)"
+format = '\[⏱ $duration\]($style)'
```

Comment:
I think this should actually be 

```suggestion
format = '\[[⏱ $duration]($style)\]'
```

Otherwise the style will not be applied correctly

Reply:
That way it will look like this:

![Screenshot_20210803_102628](https://user-images.githubusercontent.com/27806916/127984029-c2ba8c57-1d97-4d85-a2ca-65ef46fe4ebb.jpg)

Reply:
Adding a space behind duration works
``` toml
format = '\[[⏱<ICON> $duration<SPACE>]($style)\]' 
```